docs/charter.md

```diff
@@ -147,58 +150,52 @@ This means that the products of our work should be considered _best effort_, and
-- [CPAN Software Bills of Materials](https://github.com/orgs/CPAN-Security/projects/1) – Tooling for creating and verifying Software Bills of Materials for CPAN distributions
-- [CPAN Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11) – High-priority updates to CPAN packages
-- …and [other projects](https://github.com/orgs/CPAN-Security/projects)
-- ~~Registering as a CVE Numbering Authority~~
-  - **FIXME**: Probably not useful? Please share your thoughts.
-- …
+- [Artifact Transparency Logs](https://github.com/orgs/CPAN-Security/projects/2)
+  - Introduce sigstore or sigsum to CPAN, possibly based on guidelines from [transparency.dev](https://transparency.dev)
+- [Provenance and Supply Chain Security](https://github.com/orgs/CPAN-Security/projects/3)
+  - Introduce tooling and infrastructure for establishing provenance, chain-of-trust and chain-of-custody
+  - Introduce tooling for downstream verification of provenance, chain-of-trust and chain-of-custody
+- [Metadata and Software Bill of Materials](https://github.com/orgs/CPAN-Security/projects/1)
+  - Tooling for the creation and verification of SBOM objects commonly used to communicate the composition and pedigree of CPAN dependencies, even when these dependencies cross ecosystem boundraries.
+  - Update CPAN package metadata to enable end-users to comply with legislative obligations.
+- [Vulnerability Index](https://github.com/orgs/CPAN-Security/projects/10)
+  - Standardization and publishing of CPAN package vulnerabilities in relevant indexes (our own, CPANSA, CVE or whatever), to ensure that common (including third-party) tooling for Software Composition Analysis or vulnerability assessment work with CPAN dependencies.
+- [Software Composition Analysis](https://github.com/orgs/CPAN-Security/projects/6)
+  - Improve or create user-interfacing tooling used for analyzing dependencies for known vulnerabilities.
+- [Security Patch Tooling](https://github.com/orgs/CPAN-Security/projects/11)
+  - Develop tooling for publishing and applying third-party security patches to CPAN distributions that have non-responsive authors, to enable high-priority updates to CPAN packages.
+  - Explore opportunities for simplifying downstream fixes to reach upstream authors.
+- [Security Outreach & Information](https://github.com/orgs/CPAN-Security/projects/12)
+  - Establish a website and social media presence for outreach and information sharing, and keep these up to date.
+- [CPANSec Governance, Policy & Funding](https://github.com/orgs/CPAN-Security/projects/7)
+  - Establish procedures for responsible disclosure, communication and other related mechanisms, including documentation on how vulnerabilities and malicious code incidents are handled.
+  - Create a group charter and accountability procedures.
+  - Raise funds for CPAN Security Group projects.
 
 
 ### Tools
 
-- TBD
+- TODO
```

Comment:
Shouldn't we just remove this section completely and add it once we get the tuits and/or the aforementioned projects start bearing fruit?

Reply:
Fair enough.